Mithril update 2024-07-10 (#427)
IntersectMBO/cardano-updates: blog/2024-07-10-mithril.md

---
title: Mithril Team Update
slug: 2024-07-10-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team worked on the support for Cardano node version `9.0.0` and started working on a new Mithril distribution. They also continued implementing the certification of Cardano transactions in Mithril networks. They fixed some bugs blocking the REST API during transactions import, causing resource exhaustion on the aggregator Cardano node, and creating some exceptions in the explorer. They also kept redacting a draft CIP for the diffusion of Mithril signatures through the Cardano network.
> 
> Finally, the team worked on an enhancement of the artifact production in the aggregator which avoids blocking the certification and they finalized enforcing linting of all the files in the repository with the CI.

Reviewer: ```suggestion
This week, the Mithril team worked on supporting Cardano node version `9.0.0` and started working on a new Mithril distribution. They also continued implementing the certification of Cardano transactions in Mithril networks. They fixed some bugs blocking the REST API during transactions import, causing resource exhaustion on the aggregator Cardano node, and creating some exceptions in the explorer. They also kept redacting a draft CIP for the diffusion of Mithril signatures through the Cardano network.

Finally, the team worked on enhancing the artifact production in the aggregator to avoid blocking the certification and finalized enforcing the linting of all the files in the repository with the CI.
```
